pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -73,6 +73,7 @@ pipelineJob("${BUILD_FOLDER}/${JOB_NAME}") {
         stringParam('overrideFileNameVersion', "", "When forming the filename, ignore the part of the filename derived from the publishName or timestamp and override it.<br/>For instance if you set this to 'FOO' the final file name will be of the form: <code>OpenJDK8U-jre_ppc64le_linux_openj9_FOO.tar.gz</code>")
         booleanParam('cleanWorkspaceBeforeBuild', false, "Clean out the workspace before the build")
         booleanParam('propagateFailures', propagateFailures, "If true, a failure of <b>ANY</b> downstream build (but <b>NOT</b> test) will cause the whole build to fail")
+        booleanParam('keepTestReportDir', false, 'If set to true the test report dir will not be deleted. Does not apply to JUnit jobs which are always kept, eg.openjdk.')
```

Comment:
It is worth noting somewhere that we always keep the test report dir if the test job fails.  This controls whether its kept, when the test job passes.

Reply:
Ah yes good point!

Reply:
updated